src/lib/locale/formats.js

```diff
@@ -8,6 +8,10 @@ export var defaultLongDateFormat = {
 };
 
 export function longDateFormat (key) {
+    if (!key) {
```

Comment:
Why is this necessary? Who calls this method ?

Reply:
I am trying to fix this issue. while calling longDateFormat without params. 
Uppercase function 'undefind' for locale. #4386

Reply:
isSameOrAfter() function in campare.js using isSame() and isAfter(), both the function need to pass isUtc boolean value to createLocal() function in order to check the utc string format.